monai/data/wsi_reader.py

```diff
@@ -56,9 +56,11 @@ class BaseWSIReader(ImageReader):
     supported_suffixes: List[str] = []
     backend = ""
 
-    def __init__(self, level: int, **kwargs):
+    def __init__(self, level: int, channel_last: bool = False, **kwargs):
         super().__init__()
         self.level = level
+        self.channel_last = channel_last
+        self.channel_dim = -1 if self.channel_last else 0
```

Comment:
And here we verify the supported `channel_dim` values so far (like 0, -1, 2, -3).
What do you think?

Thanks.